docs/Harmonization.md

```diff
@@ -102,14 +102,15 @@ Potential threats denote a category of observations which attribute a potential
 |artifact|Artifacts refer to host-based indicators, such as checksums, file paths or detection rules.|These observations do not directly reference a compromise, rather can be used for monitoring and detection.|
 |attribution|Observations that can be attributed to malicious activity, which are not detailed enough to action on, from the victim notification perspective.|These observations require further assessment or analysis.|
 |blocked resource|Some sources provide reputation lists which clearly refer to abusive behavior (such as spamming) but fail to denote the exact reason why a given identity has been listed. The justification may be anecdotal or missing entirely.|Services appearing on these lists will have difficulty to operate normally, as their service specific communication will be blocked by third parties.|
-|compromised account|Leaked user credentials, which have been taken from a compromised online service.|The compromised credentials may lead to further unauthorized use through password re-use even if the compromised service is not part of the report recipient's infrastructure.|
-|cve|This type of observation identifies a product and a version of software, which contains a specific vulnerability.|These observations do not detail the affected host or service, rather than they can be used to identify such services especially if they are not directly exposed to the Internet or refer to client side vulnerabilities.|
+|breached data|Observations on entire data dumps that often reside on the dark web and can be associated with an organization.|The breached data dump is a sign of suspected compromise, but further analysis is needed to ascertain where the breach has taken place and who is affected by it.|
+|compromised account|Observations on leaked user credentials, which have been taken from a compromised online service.|The compromised credentials may lead to further unauthorized use through password re-use even if the compromised service is not part of the report recipient's infrastructure.|
```

Comment:
Maybe somehow rephrase "The compromised credentials may lead to further unauthorized use through password re-use even if the compromised service is not part of the report recipient's infrastructure." ... how about something like "In addition to compromising the service in question, other services used by same users are at risk due to common practice of password re-use."?

Reply:
Fair point. I modified the impact statement as follows:
> In addition to posing a risk to the breached service, other services used by the same user are at risk due to the common practice of password re-use.